include/foxy/type_traits.hpp

```diff
@@ -24,6 +24,14 @@ struct make_void
 template <class... Ts>
```

Comment:
`make_void` should be in a `detail` namespace

Reply:
#include <boost/type_traits/make_void.hpp>

Reply:
Done!